mobileapps/views.py

```diff
@@ -24,37 +25,35 @@
 from edx_solutions_api_integration.utils import get_ids_from_list_param
 from edx_solutions_organizations.models import Organization
 from edx_solutions_organizations.serializers import BasicOrganizationSerializer
-from image_helpers import get_logo_image_names, set_has_logo_image, create_logo_images
+from image_helpers import get_image_names, set_has_logo_image, create_images
 from openedx.core.djangoapps.profile_images.exceptions import ImageValidationError
 from openedx.core.djangoapps.profile_images.images import IMAGE_TYPES, validate_uploaded_image
 
-
+from mobileapps.image_helpers import get_image_names
 from mobileapps.models import MobileApp, NotificationProvider, Theme
 from mobileapps.serializers import MobileAppSerializer, NotificationProviderSerializer, ThemeSerializer
 from mobileapps.tasks import publish_mobile_apps_notifications_task
 
+_ORGANIZATION_LOGO_IMAGE_SIZES_MAP = settings.ORGANIZATION_LOGO_IMAGE_SIZES_MAP.values()
+_ORGANIZATION_HEADER_BG_IMAGE_SIZES_MAP = settings.ORGANIZATION_HEADER_BG_IMAGE_SIZES_MAP.values()
```

Comment:
can we use `settings.ORGANIZATION_HEADER_BG_IMAGE_SIZES_MAP` where needed since we are using it only once in this file?